appengine/standard/migration/memorystore/main_test.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 202 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from mock import patch
+import pytest
+import uuid
+
+import main
+
+KEY_PREFIX = str(uuid.uuid4())  # Avoid interfering with other tests
+TEST_VALUES = {
+    KEY_PREFIX + "weather_USA_98105": "raining",
+    KEY_PREFIX + "weather_USA_98115": "cloudy",
+    KEY_PREFIX + "weather_USA_94105": "foggy",
+    KEY_PREFIX + "weather_USA_94043": "sunny",
+}
+
+
+@patch('main.query_for_data', return_value='data')
+def test_get_data_not_present(query_fn, testbed):
+    try:
+        main.client.set(KEY_PREFIX + 'counter', '0', 9000)
+    except Exception:
```

Comment:
Is there a more specific exception that can be used here?

Reply:
It could be redis.RedisError, which all other Redis errors inherit from.

Reply:
The double quotes are exact lines from the App Engine standard for Python 2.7 memcache sample code. They're put there to contrast the old and new ways. This is described in the block comments in the code. I also just added mention of that to the README.md.